api/pb/android/app/src/main/java/io/textile/textileexample/MainActivity.java

```diff
@@ -5,34 +5,57 @@
 import android.view.View;
 
 import io.textile.threads.Client;
+import io.textile.threads.Config;
+import io.textile.threads.DefaultConfig;
+import io.textile.threads.TextileConfig;
 
 public class MainActivity extends AppCompatActivity {
 
     Client client;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
+        System.out.println("Info: " + "Startup");
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
-
         initIPFS();
     }
 
     public void onButtonClick(View v) {
         try {
             String storeId = client.NewStoreSync();
-            System.out.println("Success: " + storeId);
+            System.out.println("Thread Info: " + storeId);
         } catch (Exception e) {
-            System.out.println(e.getMessage());
+            System.out.println("Thread Info: " + e.getMessage());
         }
     }
 
     private void initIPFS() {
         try {
-            client = new Client("localhost", 6006);
-            client.Connect();
+            /**
+             * To use hosted threads from Textile, use the following instead of DefaultConfig
+             */
+            /*
+                Config config = new TextileConfig(
+                    PROJECT_TOKEN,
+                    DEVICE_UUID
+                );
+                // Optionally, restore a session for the same user by supplying the session ID
```

Comment:
maybe include a note here about where to find the `session ID` (e.g., after a `textile login` you'll find it in `~/.textile/auth.yml`)

Reply:
no, that's for developer accounts. for app accounts, they just need the project token `textile project add token` and then session id is generated as a result of running the above without a provided session id a first time

Reply:
noice!